src/extractors.rs

```diff
@@ -100,27 +108,27 @@ fn section<'a, I>(events: &mut I) -> Result<DocSection, ParseError> where
             DocSection::LifetimeParameters(try!(list(events, &headline))),
         "returns" =>
             DocSection::Returns(
-                md(events.take_while(not!(start Tag::List(_)))),
+                md(events.map(|(event, _)| event).take_while(not!(start Tag::List(_)))),
                 try!(list(events, &headline))),
         _ =>
             DocSection::Custom(
                 headline,
-                md(events.take_while(not!(start Tag::Header(1))))),
+                md(events.map(|(event, _)| event).take_while(not!(start Tag::Header(1))))),
```

Comment:
I'm currently sitting at the train station in Cologne and it's -3°C, so I only wrote this: https://play.rust-lang.org/?gist=cdabc6c2cac0cd069ec7522bf136983b&version=stable&backtrace=0